### ISSUE TITLE
An unexpected error occurred while using the KiCad Project Generator

### ISSUE THREAD

**Issue by nanpuhaha** [adamws/keyboard-tools#10]
An unexpected error occurred. Please try again. If the error persists, you can submit the issue with the following details at [GitHub](https://github.com/adamws/keyboard-tools/issues/new):

```python
Traceback (most recent call last):

File "/kicad/src/tasks.py", line 71, in generate_kicad_project
log_path = kicad.new_pcb(task_id, task_request, __update_percentage)
File "/kicad/src/kicad.py", line 280, in new_pcb
generate_pcb_file(project_full_path, project_name)
File "/kicad/src/kicad.py", line 122, in generate_pcb_file
raise Exception(f"Generate .kicad_pcb from netlist failed: details: {log}")
Exception: Generate .kicad_pcb from netlist failed: details: ./kicad/pcbnew/action_plugin.cpp(163): assert "PgmOrNull()" failed in register_action().
Unable to find footprint Stabilizer_Cherry_MX_2u in /kicad/c9197519-0324-4d67-b85f-e0fc0a839967/k-bd-230625/libs/keyswitch-kicad-library/footprints/Mounting_Keyboard_Stabilizer.pretty
Unable to find footprint Stabilizer_Cherry_MX_2u in /kicad/c9197519-0324-4d67-b85f-e0fc0a839967/k-bd-230625/libs/keyswitch-kicad-library/footprints/Mounting_Keyboard_Stabilizer.pretty
Unable to find footprint Stabilizer_Cherry_MX_2u in /kicad/c9197519-0324-4d67-b85f-e0fc0a839967/k-bd-230625/libs/keyswitch-kicad-library/footprints/Mounting_Keyboard_Stabilizer.pretty
Unable to find footprint Stabilizer_Cherry_MX_2u in /kicad/c9197519-0324-4d67-b85f-e0fc0a839967/k-bd-230625/libs/keyswitch-kicad-library/footprints/Mounting_Keyboard_Stabilizer.pretty
```

**Comment by adamws** [comment]
can you share layout file and settings you used?

**Comment by adamws** [comment]
should be fixed by release v0.3
